core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -924,6 +924,21 @@ private static RelDataType arrayReturnType(SqlOperatorBinding opBinding) {
           OperandTypes.STRING.or(OperandTypes.BINARY),
           SqlFunctionCategory.STRING);
 
+  @LibraryOperator(libraries = {BIG_QUERY})
+  public static final SqlFunction FROM_BASE32 =
+      SqlBasicFunction.create("FROM_BASE32",
+          ReturnTypes.explicit(SqlTypeName.VARBINARY)
+              .andThen(SqlTypeTransforms.TO_NULLABLE),
+          OperandTypes.STRING, SqlFunctionCategory.STRING);
+
+  @LibraryOperator(libraries = {BIG_QUERY})
+  public static final SqlFunction TO_BASE32 =
+      SqlBasicFunction.create("TO_BASE32",
+          ReturnTypes.explicit(SqlTypeName.VARCHAR)
```

Comment:
If there's multiple functions using this return type with this transformation applied, is it worth adding it's own entry to ReturnTypes?

Reply:
Good idea, and I have added in [CALCITE-5728](https://github.com/apache/calcite/pull/3237/files), maybe I could rebase and modify it after [CALCITE-5728](https://github.com/apache/calcite/pull/3237/files) is merged

Reply:
@tanclary I have updated this PR, please take a look when you have time